usaspending_api/etl/submission_loader_helpers/file_c.py

```diff
@@ -171,23 +171,6 @@ def _save_file_c_rows(certified_award_financial, total_rows, start_time, skipped
             update_skipped_tas(row, tas_rendering_label, skipped_tas)
             continue
 
-        # Find a matching transaction record, so we can use its subtier agency information to match to (or create) an
-        # Award record.
-
-        # Find the award that this award transaction belongs to. If it doesn't exist, create it.
-        filters = {}
-        if row.get("piid"):
-            filters["piid"] = row.get("piid")
-            filters["parent_piid"] = row.get("parent_award_id")
-        else:
-            if row.get("fain") and not row.get("uri"):
-                filters["fain"] = row.get("fain")
-            elif row.get("uri") and not row.get("fain"):
-                filters["uri"] = row.get("uri")
-            else:
-                filters["fain"] = row.get("fain")
-                filters["uri"] = row.get("uri")
-
```

Comment:
We able to get rid of this because we do the C-to-D linkage from SQL now at the end of the load?

Was this just accidentally left in here as dead code? -- i'm not seeing where these filters dict would have been used.

Reply:
dead code